### ISSUE TITLE
Explain or link to hardware design to work with integration

### ISSUE THREAD

**Issue by BramVerheij** [golles/ha-kamstrup_403#3]
**Is your feature request related to a problem? Please describe.**
I would love to use the integration, but I am still searching for the hardware requirements or implementation to make it work.

**Describe the solution you'd like**
Describe the hardware solution in the Readme or link to applicable forum post would help.

**Describe alternatives you've considered**
I have checked the link to the forum post, but it does not link to a particular forum topic where I could find more information.

**Additional context**
Would love to be able to use this!

**Comment by golles** [comment]
You are right, I've never added that to this readme, unlike what I did with the component for my previous meter, please refer to that one for now, until I manage to add it here: 

https://github.com/golles/Home-Assistant-Sensor-MC66C
